protocol/contracts/v2/TempleDebtToken.sol

```diff
@@ -231,8 +231,10 @@ contract TempleDebtToken is ITempleDebtToken, TempleElevatedAccess {
             }
         }
 
-        emit Transfer(_debtor, address(0), _burnAmount);
-        _burn(debtor, _burnAmount, _totalPrincipalAndBase);
+        if (_burnAmount > 0) {
```

Comment:
Addresses the actual issue https://github.com/yAudit/temple-lending-report/issues/54